packages/types/src/plugins.ts

```diff
@@ -86,6 +86,7 @@ export type PluginStatusRedux = {
     version: string;
     active: boolean;
     state: number;
+    error?: string;
```

Comment:
I'm not sure if this is the right way to type this. It's technically always defined, but is usually the empty string. I marked it as optional to show that it's not necessarily expected to have a meaningful value